AGENTS.md

```
@@ -0,0 +1,220 @@
+# Run tests
+pytest tests/
+
+# Serve documentation locally (requires docsify-cli)
+cd docs && docsify serve
+```
+
+See `pyproject.toml` for the full list of optional dependency groups.
+
+## Code Organization
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)